utility/src/main/java/org/oppia/android/util/parser/html/LiTagHandler.kt

```diff
@@ -0,0 +1,173 @@
+package org.oppia.android.util.parser.html
+
+import android.content.Context
+import android.text.Editable
+import android.text.Spannable
+import android.text.Spanned
+import org.oppia.android.util.locale.OppiaLocale
+import java.util.Stack
+import javax.inject.Inject
+
+/** The custom <li> tag corresponding to [LiTagHandler]. */
+const val CUSTOM_LIST_LI_TAG = "oppia-li"
+
+/** The custom <ul> tag corresponding to [LiTagHandler]. */
+const val CUSTOM_LIST_UL_TAG = "oppia-ul"
+
+/** The custom <ol> tag corresponding to [LiTagHandler]. */
+const val CUSTOM_LIST_OL_TAG = "oppia-ol"
+
+/**
+ * A custom tag handler for properly formatting bullet items in HTML parsed with
+ * [CustomHtmlContentHandler].
+ */
+class LiTagHandler @Inject constructor(
+  private val context: Context,
+  private val displayLocale: OppiaLocale.DisplayLocale
+) :
+  CustomHtmlContentHandler.CustomTagHandler {
+
+  override fun handleOpeningTag(
+    output: Editable,
+    tag: String,
+    lists: Stack<CustomHtmlContentHandler.ListTag>
+  ) {
+    when (tag) {
+      CUSTOM_LIST_UL_TAG ->
+        lists.push(Ul(context, tag))
+
+      CUSTOM_LIST_OL_TAG ->
+        lists.push(Ol(context, tag, displayLocale))
+
+      CUSTOM_LIST_LI_TAG ->
+        lists.peek().openItem(output)
```

Comment:
```suggestion
      CUSTOM_LIST_UL_TAG -> lists.push(Ul(context, tag))
      CUSTOM_LIST_OL_TAG -> lists.push(Ol(context, tag, displayLocale))
      CUSTOM_LIST_LI_TAG -> lists.peek().openItem(output)
```

Some whitespace shortening for conciseness.

Please also update the ``while`` in the next method to match.